src/styles/less/index.less

```diff
@@ -40,8 +40,6 @@
 @import 'GalleryView';
 @import 'EditPlayers';
 @import 'NewChildCard';
-@import 'AudioBook';
```

Comment:
I believe this is a component done by one of the other teammates so we need to make sure the import is still there.

Reply:
I forgot to put it back for the pr. While trying to develop it would not let me proceed. It said it had an issue with react-audio or something like that. I reinstalled node modules and it did the samething. Does it work for you with AudioBook?